cmd/podman/cp.go

```diff
@@ -19,7 +19,6 @@ import (
 	"github.com/containers/storage/pkg/archive"
 	"github.com/containers/storage/pkg/chrootarchive"
 	"github.com/containers/storage/pkg/idtools"
-	securejoin "github.com/cyphar/filepath-securejoin"
```

Comment:
Why are you dropping securejoin?  I thought that was the prefered way to handle directory joins especially if a symlink was involved?

Reply:
@TomSweeneyRedHat I dropped all calls that had hidden evals of the symlinks as that caused the initial issue.  We need to eval once.

Reply:
Sure, the re-evaluation is a problem, but that’s not enough of a justification to just discard all pre-existing protections (… without even mentioning that in the PR / commit description, even).

This PR currently replaces a vulnerability in a corner case with an even easier-to-hit vulnerability.